src/test/java/core/basesyntax/service/ParseFileImplTest.java

```diff
@@ -0,0 +1,47 @@
+package core.basesyntax.service;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.service.impl.ParseFileImpl;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.After;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class ParseFileImplTest {
+    private static final String HEAD_LINE = "type,fruit,quantity";
+    private static final String SECOND_LINE = "p,apple,30";
+    private static final String CURRENT_VALUE = "p,apple,30";
+    private static ParseFile parser;
+    private static List<String> EMPTY_LIST;
+    private static List<String> dataList;
+    private static List<String> current;
+
+    @BeforeClass
+    public static void beforeClass() {
+        parser = new ParseFileImpl();
+        EMPTY_LIST = new ArrayList<>();
+        dataList = new ArrayList<>();
+        dataList.add(HEAD_LINE);
+        dataList.add(SECOND_LINE);
+        current = new ArrayList<>();
+    }
+
+    @After
+    public void tearDown() {
+        EMPTY_LIST.clear();
+    }
+
+    @Test(expected = RuntimeException.class)
```

Comment:
you haven’t corresponding checks in your solution. So now you are testing system exceptions. Add checking to your solution or remove this test case (check it everywhere)